docs/tasks.md

```diff
@@ -1,5 +1,21 @@
 # What can it do?
 
+List of all the keys under `config.tasks`:
+
+| Name                                                                                       | Key           |
+| ------------------------------------------------------------------------------------------ | ------------- |
+| [Add Repositories](#add-repositories)                                                      | `key`         |
+| [Add personal package archives (PPA)](#add-personal-package-archives-ppa)                  | `ppa`         |
+| [Install apt packages](#install-apt-packages)                                              | `apt`         |
+| [Install Debian package archives (.deb files)](#install-debian-package-archives-deb-files) | `dev`         |
+| [Install Static binaries to /usr/local/bin](#install-static-binaries-to-usrlocalbin)       | `binaries`    |
+| [Install python packages (via pip)](#install-python-packages-via-pip)                      | `pip2`/`pip3` |
```

Comment:
Installing Python packages will be removed in v7.0 and above. Delete this line

Reply:
Can you share why do you removing the python pip packages installation?

Reply:
Almost always installing pip packages system-wide is a bad idea. They do not get automatic security updates almost always cause issues with dependencies. Use `poetry` or python's `venv`. In most cases python packages should be installed in virtual environments Or  with `--user`

Reply:
Mmmmm.... I'm considering the `pip` not like packages to use in development, but a way to install CLI tools.

For example, I not writing in Python these days, but today I installed the `youtube-dl` through `pip` because it is easier.

Of course I removed the row since it does not exist in the current version.